Document SF_DEFAULT_USER and SF_DEFAULT_PASSWORD (#64)
localstack/snowflake-docs: content/en/references/configuration/index.md

| `DNS_NAME_PATTERNS_TO_RESOLVE_UPSTREAM` | `*.s3.amazonaws.com` (example) | List of domain names that should NOT be resolved to the LocalStack container, but instead always forwarded to the upstream resolver (S3 for example). this would be required when importing data into a stage from an external S3 bucket on the real AWS cloud. Comma-separated list of Python-flavored regex patterns. |
| `SF_HOSTNAME_REGEX` | `snowflake\..+` (default) | Allows you to customize the hostname used for matching the Snowflake API routes in the HTTP router. If not set, then it matches on any hostnames that contain a `snowflake.*` subdomain (e.g., `snowflake.localhost.localstack.cloud`). |
| `SF_CSV_IMPORT_MAX_ROWS` | `50000` (default) | Maximum number of rows to import from CSV files into tables |

> | `SF_DEFAULT_USER` | `test` (default) | Change the default user to use in the Snowflake emulator. |
> | `SF_DEFAULT_PASSWORD` | `test` (default) | Change the default password to use in the Snowflake emulator. |

Reviewer: ```suggestion
| `SF_DEFAULT_USER` | `test` (default) | Specify the default user to be used by the Snowflake emulator. |
| `SF_DEFAULT_PASSWORD` | `test` (default) | Specify the default password to be used by the Snowflake emulator. |
```
Just a nitpick wording suggestion!
